data/coalition/coalition culture conversations.txt

```diff
@@ -233,3 +233,137 @@ mission "Coalition Folklore: Kimek"
 				decline
 
 
+
+mission "Coalition Departing Trouble - A"
+	minor
+	invisible
+	to offer
+		has "Coalition: First Contact: done"
+		random < 2
+	source
+		government "Coalition"
+		not attributes "coalition station"
+	destination "Ring of Friendship"
```

Comment:
Would it make sense to fail these missions after the `on enter`? As I understand it, they would be in your save file until you land on the ring, even though they don't really do anything anymore... (I might be mistaken, though.)

Reply:
Removed destination and added that.